pages.zh/linux/flatpak.md

```diff
@@ -0,0 +1,27 @@
+# flatpak
+
+> 构建、安装和运行 Flatpak 应用和运行时。
+
+- 运行已安装应用：
+
+`flatpak run {{name}}`
```

Comment:
Please translate the arguments for get this easier for non-English speakers (in all examples).